QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -297,10 +261,10 @@ def convert_to_cloud_kvstyle(
         inputs = tokenizer(input_str, return_tensors="pt")
 
     try:
-        pt_outputs = model(**inputs)
+        pt_outputs = transformed_model(**inputs)
         output_names = list(pt_outputs.keys())
     except Exception as e:
-        print(f"Model {model_name} Execution failed in pytorch:%s", e)
+        print(f"Model {transformed_model.__class__,__name__} Execution failed in pytorch:%s", e)
 
     # Raise error if expected outputs are not present
     assert "logits" in output_names, "logits not found in output"
```

Comment:
This error might be misleading if failure in execution in torch is due to any other reason

Reply:
I want to remove the try/except block here, If it fails we can't certainly know if it failed because of our changes or maybe user passed wrong input or any other reason..
Let pytorch give the error, as we can't figure out why exactly it failed in qeff..

Reply:
Yes, Agree! We can allow it to break at the modeling side from pytorch pov to get the exact issue.

Reply:
Removed try-except